client-cli/src/command/wallet_command.rs

```diff
@@ -56,17 +56,20 @@ impl WalletCommand {
                 "Passphrases do not match",
             ));
         }
+        let enckey = derive_enckey(&passphrase, name).err_kind(ErrorKind::InvalidInput, || {
```

Comment:
Add a CLI command to get HMAC from `name` and `passphrase`.

Reply:
Done, named `auth_token`.